src/generateNumber.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const generateNumber = () => {
+  let result = '';
+
+  while (result.length < 4) {
```

Comment:
When you use some static values like 4 here especially when you use it in two or more places, it's better to move it to some variable.